dlp/api/Snippets/InspectImageForSensitiveDataWithInfoTypes.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// [START dlp_inspect_image_all_infotypes]
+
+using System;
+using System.IO;
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Dlp.V2;
+using Google.Protobuf;
+
+public class InspectImageForSensitiveDataWithInfoTypes
+{
+    public static InspectContentResponse InspectImage(
+        string projectId,
+        string filePath)
+    {
+        // Instantiate dlp client.
+        var dlp = DlpServiceClient.Create();
+
+        // Construct the content item.
+        var contentItem = new ContentItem
+        {
+            ByteItem = new ByteContentItem
+            {
+                Type = ByteContentItem.Types.BytesType.ImagePng,
+                Data = ByteString.FromStream(new FileStream(filePath, FileMode.Open))
+            }
+        };
+
+        // Construct the request.
+        var request = new InspectContentRequest
+        {
+            ParentAsLocationName = new LocationName(projectId, "global"),
+            Item = contentItem
+        };
+
+        // Call the API.
+        var response = dlp.InspectContent(request);
+
+        // Inspect the response.
+        var resultFindings = response.Result.Findings;
+
+        Console.WriteLine($"Findings: {resultFindings.Count}");
+
+        foreach (var f in resultFindings)
```

Comment:
Better output formatting?

Instead of producing output like:

```
Findings:
    Quote: q1
    InfoType: i1
    Likelihood: l1
    Quote: q2
    InfoType: i2
    Likelihood: l2
    Quote: q3
    InfoType: i3
    Likelihood: l3
```

can we add some spacing between entries? And maybe only indent quote and likelihood, so that the output looks more like:

```
Findings:

InfoType: i1
    Quote: q1
    Likelihood: l1

InfoType: i2
    Quote: q2
    Likelihood: l2

InfoType: i3
    Quote: q3
    Likelihood: l3
```


Also since this is an image, could you add co-ordinates as well to demonstrate that we provide that information? That would be `f.location.contentLocations[*].imageLocation.boundingBoxes[*]`. (There can be multiple results for a single infotype, hence arrays).

Ideally we could flatten them and present them as one more item in each finding, making it something like:

```
Findings:

InfoType: i1
    Quote: q1
    ImageLocations: [(left1,top1,+width1,+height1), ...] 
    Likelihood: l1

...
```

Reply:
Regarding the output format, All the code samples follow same output format (pre-written); So, Just to be consistent about it, I have kept the output format like that.

Reply:
![image](https://user-images.githubusercontent.com/77317274/233552700-bd2c8f3e-d50c-4981-ac02-88253f2a6553.png)
this kind of output of ImageLocations you were asking?

Reply:
Sure, that format works too.

I don't know where we got the format from, but going forward we should make the output easier for users to understand. The lack of spacing makes it difficult to understand that VeryLikely is associated with PHONE_NUMBER and not PERSON_NAME...

Reply:
Okay. Let me change the format for this example.

Reply:
Done.